### PR TITLE
BXMSDOC-6808-main: Add optional verification step and link to related procedure

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 1]
```diff
@@ -135,7 +135,9 @@ The following example shows how to configure a for {CENTRAL} instance:
 
 --
 +
-. Send a GET request to `http://_SERVER:PORT_/kie-server/services/rest/server/` to verify that {KIE_SERVER} started successfully. After authentication, you receive an XML response similar to the following example:
+. Optionally, once {KIE_SERVER} has started, you can send a GET request to `http://_SERVER:PORT_/kie-server/services/rest/server/` to verify that {KIE_SERVER} started successfully. For more information about running {PRODUCT} on {KIE_SERVER}, see {URL_INSTALLING_AND_CONFIGURING}#eap-ba-dm-run-proc_install-on-eap[_Running {PRODUCT}_].
```
Comment (briandooley) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:138]:
```suggestion
. When {KIE_SERVER} starts, you can send a GET request to `http://_SERVER:PORT_/kie-server/services/rest/server/` to verify that {KIE_SERVER} has started successfully. For more information about running {PRODUCT} on {KIE_SERVER}, see {URL_INSTALLING_AND_CONFIGURING}#eap-ba-dm-run-proc_install-on-eap[_Running {PRODUCT}_].
```
Removed "optionally" as saying "you _can_ send" implies optionality.

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc [hunk 2]
```diff
@@ -135,7 +135,9 @@ The following example shows how to configure a for {CENTRAL} instance:
...
++
+After authentication, you receive an XML response similar to the following example:
```
Comment (briandooley) [inline doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:140]:
```suggestion
After successful authentication, you receive an XML response similar to the following example:
```

**Review by briandooley — APPROVED** [review]
Have made a few suggestions but no biggies. Feel free to use or not.

File: doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc (hunk 1 above)
Comment (briandooley) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:138]: I'm ok with "To verify, you can".

As for passive voice, wasn't sure if "once {KIE_SERVER} has started," strictly passed the "by zombies" test. I.e. "once {KIE_SERVER} has started by zombies" isn't correct. I would say remove passive voice if possible without undoing the meaning of what you are trying to say. Guess that doesn't help much. See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice for more about passive voice.

Comment (csherrar) [reply doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc:138]: Thanks Brian

**Pull request merged**